docs/contributing/2.-coding-standard.md

```diff
@@ -30,22 +30,17 @@ We use a .editorconfig file to define this.
 
 Naming things is hard.
 {{ project.name }} has a few strict guidelines on the usage of variable names, which hopefully will reduce some of the guesswork:
+
 - No one character variable names.
-    - Except for x, y, and z as coordinates.
+  - Except for x, y, and z as coordinates.
 - It's not okay to override built-in functions.
-    - Except for `id`. Guido himself thought that shouldn't have been moved to the system module. It's too commonly used, and alternatives feel very artificial.
+  - Except for `id`. Guido himself thought that shouldn't have been moved to the system module. It's too commonly used, and alternatives feel very artificial.
```

Comment:
Maybe a reference link

Reply:
I think this is not relevant, I almost deleted it. Do you mind if I do?